tests/src/Kernel/BclIconPathOverrideTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel;
+
+use Drupal\Core\Theme\ThemeManagerInterface;
+use Drupal\KernelTests\KernelTestBase;
+
+/**
+ * Tests the bcl_icon_path theme override functionality.
+ */
+class BclIconPathOverrideTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'system',
+  ];
+
+  /**
+   * The theme manager service.
+   *
+   * @var \Drupal\Core\Theme\ThemeManagerInterface
+   */
+  protected ThemeManagerInterface $themeManager;
+
+  /**
+   * Set up the test environment.
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    // Initialize the theme manager service.
+    $this->themeManager = $this->container->get('theme.manager');
+
+    // Install the theme. Subsequently, this installs also the base themes.
+    $this->container->get('theme_installer')->install(['oe_bootstrap_theme', 'oe_bootstrap_theme_test_subtheme']);
+    $this->config('system.theme')->set('default', 'oe_bootstrap_theme_test_subtheme')->save();
```

Comment:
There's no need to do this here.

Reply:
yep, refactored the tests with data provider